server/broadcast.go

```diff
@@ -359,7 +363,7 @@ func (bsm *BroadcastSessionsManager) isVerificationEnabled() bool {
 	return bsm.VerificationFreq > 0
 }
 
-func (bsm *BroadcastSessionsManager) shouldRunVerification(sessions []*BroadcastSession) bool {
+func (bsm *BroadcastSessionsManager) shouldSkipVerification(sessions []*BroadcastSession) bool {
 	if bsm.verifiedSession == nil {
```

Comment:
So it's now saying we should skip verification if there isn't a verified session?

Reply:
It says we should *not* skip verification (return false), if there's no verified session. Previous method name was confusing.